lib/const.rb

```diff
@@ -220,5 +220,6 @@
 
 CREW_ESSENTIAL_FILES = `LD_TRACE_LOADED_OBJECTS=1 #{CREW_PREFIX}/bin/ruby`.scan(/\t([^ ]+)/).flatten +
                        `LD_TRACE_LOADED_OBJECTS=1 #{CREW_PREFIX}/bin/rsync`.scan(/\t([^ ]+)/).flatten +
+                       `LD_TRACE_LOADED_OBJECTS=1 #{CREW_PREFIX}/bin/rdfind`.scan(/\t([^ ]+)/).flatten +
```

Comment:
The `rdfind` command is provided by `buildessential`, maybe move this check to `crew`? (check when `rdfind` is exist and being called)

Reply:
`rdfind` is used during package installs. without this bit, during a gcc11 install there is an error about a `libstdc++.so.6` missing, which will happen if any package has ever been built, because then rdfind is installed.

(rdfind is also only ~ 80k)

Reply:
Actually... no I think you are right... let me just add the `libstdc++.so.6` dep to CREW_ESSENTIAL FILES.